src/pocketmine/block/Bed.php

```diff
@@ -86,15 +87,15 @@ public function onActivate(Item $item, Player $player = null){
 				$b = $blockWest;
 			}else{
 				if($player instanceof Player){
-					$player->sendMessage(TextFormat::GRAY . "This bed is incomplete");
+					$player->sendMessage(new TranslationContainer(TextFormat::GRAY . "%tile.bed.notValid"));
```

Comment:
In the game files, this is actually "Your home bed was missing or obstructed". I don't think this is correct.